packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -359,3 +368,57 @@ export type GPCRevealedClaims = {
    */
   watermark?: PODValue;
 };
+
+/**
+ * General-purpose circuit requirements for a given proof configuration.
+ * These values will be appropriately checked against the circuit configurations
+ * at our disposal in order to accommodate all of the proof inputs. For the
+ * objects, entries, Merkle tree depth and membership list sizes, this amounts
+ * to picking the circuit description whose corresponding parameters exceed
+ * these numbers, while the choice of tuple parameters is more involved.
+ */
+export type GPCRequirements = {
+  /**
+   * Number of POD objects which must be included in a proof.
+   */
+  nObjects: number;
+
+  /**
+   * Number of POD entries which can be included in a proof.
+   */
+  nEntries: number;
+
+  /**
+   * Depth of POD merkle tree.  Max entries in any object is 2^(depth-1).
+   */
+  merkleMaxDepth: number;
+
+  /**
+   * Minimum required number of entries in each membership list to be included in proof.
```

Comment:
It's unclear whether "Minimum" or "Maximum" is the clearest word here.  Maybe leave it out entirely and start the sentence with "Number of entries..." if your intent is that this list contains the precise lengths of the inputs?  Else add another sentence to explain if this is allowed to be larger than the input.

Reply:
Also specify explicitly whether order is important here.